src/services/api/contract_classes/deprecated_contract_class.rs

```diff
@@ -393,4 +412,22 @@ mod tests {
 
         res.expect("should be able to read file");
     }
+
+    #[test]
+    fn test_from_program_json_and_class_hash_should_equal_from_path() {
+        let path = "starknet_programs/fibonacci.json";
+        let contract_class_from_path =
+            ContractClass::from_path(path).expect("should be able to read file");
+        let program_json = fs::read_to_string(path).expect("should be able to read file");
```

Comment:
It may be more practical to simply use `include_str` or `include_bytes` here.

Reply:
True! I will change this